lib/events.js

```diff
@@ -0,0 +1,207 @@
+'use strict';
+
+var _ = require('lodash');
+var joi = require('joi');
+
+exports.register = function (server, options, next) {
+  options = _.extend({ basePath: '/api/2.0' }, options);
+  var handlers = require('./handlers.js')(server, 'cd-events');
+
+  server.route([{
+    method: 'POST',
+    path: options.basePath + '/events/save',
+    handler: validateEvent('saveEvent')
+  }, {
+    method: 'GET',
+    path: options.basePath + '/events/{id}',
+    handler: handlers.actHandler('getEvent', 'id')
+  }, {
+    method: 'POST',
+    path: options.basePath + '/listEvents',
```

Comment:
the other way around: path should be `/events` and handler should be `listEvents`